app/views/admin/index.scala.html

```diff
@@ -950,6 +946,14 @@ <h1>Label Search</h1>
                 debug: false
             }, function(err, t) {
                 window.admin = Admin(_, $);
+                
+                // Load street edge data from the backend & make the loader finish after that data loads.
+                window.admin.loadStreetEdgeData().then(function() {
```

Comment:
Is it possible to move this to Admin.js? I kinda don't like that any of this stuff is in the template scala tbh :joy: